editors/vile/Portfile

```diff
@@ -3,8 +3,8 @@
 PortSystem          1.0
 
 name                vile
-version             9.8
-revision            2
+version             9.8u
+revision            1
```

Comment:
Revision should be reset to 0 for a new port version

Reply:
Done.